assets/src/edit-story/masks/frame.js

```diff
@@ -63,6 +63,26 @@ const DropTargetPath = styled.path`
   transition: opacity 0.5s;
   pointer-events: visibleStroke;
   opacity: ${({ active }) => (active ? 0.3 : 0)};
+  stroke: ${({ theme }) => theme.colors.border.selection};
+`;
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+
+  ::before {
+    transition: opacity 0.5s;
+    opacity: ${({ hasBackgroundOutline }) => (hasBackgroundOutline ? 1 : 0)};
+    display: block;
+    content: '';
+    position: absolute;
+    width: calc(100% + 8px);
+    height: calc(100% + 8px);
+    border-radius: 8px;
+    left: -4px;
+    top: -4px;
+    border: 1px solid ${({ theme }) => theme.colors.border.selection};
```

Comment:
We should make sure to align this between this PR and #6250 once either is merged. Maybe move to some shared CSS helper and/or constants.

Reply:
@barklund Is there Figma for it as well? Sam told me that the drop target colors won't change, want to confirm if this has been reconsidered meanwhile.

Reply:
Checking in [this thread](https://xwp.slack.com/archives/C01ESBB8ZN1/p1613421825239100).